control/veda_heal.py

```diff
@@ -62,36 +62,44 @@ def send_encodes(self):
         for v in self.video_query:
             encode_list = self.determine_fault(video_object=v)
             # Using the 'Video Proto' Model
-            if self.val_status is not None:
-                # Update to VAL is also happening for those videos which are already marked complete,
-                # All these retries are for the data-parity between VAL and VEDA, as calls to VAL api are
-                # unreliable and times out. For a completed Video, VEDA heal will keep doing this unless
-                # the Video is old enough and escapes from the time-span that HEAL is picking up on.
-                # cc Greg Martin
-                VAC = VALAPICall(
+            # Update to VAL is also happening for those videos which are already marked complete,
+            # All these retries are for the data-parity between VAL and VEDA, as calls to VAL api are
+            # unreliable and times out. For a completed Video, VEDA heal will keep doing this unless
+            # the Video is old enough and escapes from the time-span that HEAL is picking up on.
+            # cc Greg Martin
+            if not self.val_status and len(encode_list) > 0:
+                self.val_status = 'transcode_queue'
```

Comment:
`self.val_status` in this clause is not used in rest of the flow. Can we remove this clause?